lib/services/achievements-service.ts

```
@@ -260,33 +262,37 @@ export class AchievementsService extends BaseService {
 
         case 'saving_streak':
+        case 'consecutive-days':
           // This would require more complex logic to track saving streaks
           // For now, we'll use a simple implementation
           currentProgress = userStats.savingsTransactions;
-          shouldUnlock = currentProgress >= achievement.requirementValue;
+          shouldUnlock = currentProgress >= (typeof achievement.requirement === 'object' ? achievement.requirement.value : 0);
           break;
 
         case 'budget_adherence':
+        case 'custom':
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)